app/bundles/PluginBundle/Config/config.php

```diff
@@ -11,37 +11,42 @@
     'routes'   => array(
         'main' => array(
             // @deprecated 1.1.4 to be removed in 2.0
-            'mautic_integration_auth_callback_bc' => array(
-                'path'       => '/addon/integrations/authcallback/{integration}',
```

Comment:
While testing I realized that we are going to have to leave these three routes in main for BC (all my callbacks registered in the apps contain /s/). We have to name them something else like appending `mautic_integration_auth_callback_bc_secure` or something but we need to keep the paths and controller listed in `main` in addition to `public` so that the URLs still work.